src/pages/deriv-go/index.tsx

```diff
@@ -46,59 +57,44 @@ const items = [
     },
 ]
 
-const derivGoPortalData = {
+type DerivGoPortalType = {
+    paragraph: ReactElement
+    frame: string
+    link: string
+}
+
+const derivGoPortalData: DerivGoPortalType = {
     paragraph: (
         <Localize translate_text="Take a look at Deriv GO’s product roadmap, give us your feedback on what we’re building, and suggestions on what to build next." />
     ),
     frame: 'https://portal.productboard.com/gfueayjjwpmfhdysrrn3n3wn?hide_header=1',
     link: 'https://portal.productboard.com/gfueayjjwpmfhdysrrn3n3wn',
 }
+
 const DerivGo = () => {
     const { is_row } = useCountryRule()
-    const [is_loaded, setLoaded] = useState(false)
-
-    useEffect(() => {
-        setLoaded(true)
-    }, [useCountryRule])
-
-    if (is_loaded) {
-        if (is_row) {
-            return (
-                <Layout>
-                    <SEO
-                        title={localize(
-                            'Trade forex, synthetics, and cryptocurrencies with our app — Deriv GO.',
-                        )}
-                        description={localize(
-                            'Trade forex, synthetic indices, and cryptocurrencies wherever, whenever you want and maximise your potential profit with multipliers on Deriv GO.',
-                        )}
-                    />
-                    <Banner />
-                    <WhatIsDerivGo />
-                    <WhyTradeDerivGo />
-                    <DerivGoContent reverse P2P={items} />
-                    <StartDerivGo />
-                    <FooterBanner />
-                    <OtherApps />
-                    <Roadmap portal={derivGoPortalData} />
-                </Layout>
-            )
-        }
-
-        return <PageNotFound />
-    }
 
-    return (
-        <>
+    return is_row ? (
```

Comment:
Can we keep the condition outside of return?
like:
```
if (is_row) {
return <Layout> 
...
}
return <PageNotFound />
```

Reply:
Resolved, @mohammad-rework 👍